build.zig

```diff
@@ -27,16 +27,28 @@ pub fn build(b: *Builder) !void {
     });
     b.default_step.dependOn(&fmt_step.step);
 
+    const test_enum = enum {
+        UNIT,
+        ALL_RUNTIME,
+        NORMAL,
+        PANIC,
+    };
```

Comment:
I think these names could be more clear and perhaps add a comment above each describing what they mean.

Reply:
Yep :)